BE/sidedish11/src/main/java/dev/codesquad/java/sidedish11/controller/ApiDetailController.java

```diff
@@ -0,0 +1,27 @@
+package dev.codesquad.java.sidedish11.controller;
+
+import dev.codesquad.java.sidedish11.dto.ItemDetailResponse;
+import dev.codesquad.java.sidedish11.service.ItemService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/detail")
+public class ApiDetailController {
+    private Logger logger = LoggerFactory.getLogger(ApiDetailController.class);
+
+    @Autowired
+    private ItemService itemService;
+
+    @GetMapping("/{hash}")
+    public ResponseEntity viewItem(@PathVariable String hash) {
+        ItemDetailResponse itemDetailResponse = itemService.getItemDetail(hash);
+        return ResponseEntity.ok(itemDetailResponse);
```

Comment:
예외 처리나 기타 로직이 필요없다면, 아래와 같이 한 줄로 표현할 수도 있겠네요. 

```suggestion
        return ResponseEntity.ok(itemService.getItemDetail(hash));
```

Reply:
넵. 컨트롤러에서 응답할 때 통일성을 주고싶어서 별개로 객체 만들었습니다.